src/ui-kit/experimental/hierarchical/hierarchical-tree-grid/hierarchical-tree-grid.component.html

```diff
@@ -1,7 +1,12 @@
-<div class="example-container usa-grid">
-  <ng-container *ngTemplateOutlet="gridTemplate"></ng-container>
+<div class="example-container usa-grid" >
+  <ng-container *ngTemplateOutlet="gridData.length===0?noResults: gridTemplate"></ng-container>
```

Comment:
@thomastighe Can you add some whitespace between your operators for readability?

Reply:
added some spaces